core/src/main/java/org/infinispan/commands/functional/WriteOnlyKeyCommand.java

```diff
@@ -44,6 +54,8 @@ public void writeTo(ObjectOutput output) throws IOException {
       Params.writeObject(output, params);
       output.writeLong(FlagBitSets.copyWithoutRemotableFlags(getFlagsBitSet()));
       CommandInvocationId.writeTo(output, commandInvocationId);
+      output.writeObject(keyDataConversion);
```

Comment:
IIUC the keyDataConversion is not used for write-only commands, right? Though, I'd like to add the `key()` method to the `WriteEntryView` eventually...

Reply:
Yes it is not, I was thinking about it just right now, that the dependency to the keyDataConverter may be removed. But if you want to add the key method, in that case both are necessary. I can leave that and add the method

Reply:
Don't mix changes in one PR, unless necessary, please... just remove it here for now.

Reply:
Well, since it's a small change I don't see the harm of adding the key method. but ok, this is not a big deal for me.

Reply:
@rvansa after looking this, the keyconverter is in the abstract class that is used in several parts. I'm going to leave it as it is, considering key method will be added just after on WriteOnly entry views